### PR TITLE
CO-1559 merge skeleton

### PR DESCRIPTION
**Pull request by Ruefa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Financial Index API ![version](https://img.shields.io/badge/version-v1-blue.svg) [![openapi](https://img.shields.io/badge/openapi-2.0-green.svg)](./openapi.yaml) ![node](https://img.shields.io/badge/node-10.13-brightgreen.svg) ![npm](https://img.shields.io/badge/npm-6.11.1-orange.svg)
+# Express API Skeleton ![version](https://img.shields.io/badge/version-v1-blue.svg) [![openapi](https://img.shields.io/badge/openapi-2.0-green.svg)](./openapi.yaml) ![node](https://img.shields.io/badge/node-10.17-brightgreen.svg) ![npm](https://img.shields.io/badge/npm-6.11.1-orange.svg)
```
Comment (tsoliangwu0130) [inline README.md]: Should be `Financial Index API`.

File: README.md [hunk 2]
```diff
@@ -27,6 +27,8 @@ API to validate financial account index codes, which are used for purchasing. AP
     | `${API_USER}` | The HTTP Basic username used to authenticate API calls. |
     | `${API_PASSWD}` | The HTTP Basic password used to authenticate API calls. |
 
+5 Copy [db/mock-data-example.json](db/mock-data-example.yaml) to `db/mock-data.json`. This will serve as the JSON DB, which is not committed to source code as it will change as the POST endpoint is used.
+
```
Comment (tsoliangwu0130) [inline README.md]: This can be removed.

File: openapi.yaml [hunk 3]
```diff
@@ -124,7 +121,6 @@ paths:
           description: Activity code.
           type: string
           required: true
-        - $ref: '#/parameters/authorization'
```
Comment (tsoliangwu0130) [inline openapi.yaml]: I think this part is missing: https://github.com/osu-mist/express-api-skeleton/blob/a72e05b8b43af1ee253f142f74d010bae88b73df/openapi.yaml#L20-L26

Can you add it?

File: openapi.yaml [hunk 4]
```diff
@@ -19,6 +19,11 @@ externalDocs:
   url: https://github.com/osu-mist/financial-index-api
 produces:
   - application/json
+consumes:
+  - application/json
+security:
+  - OAuth2:
+      - full
```
Comment (tsoliangwu0130) [inline openapi.yaml]: Looks like the openapi file is broken now. Can you fix it?

Comment (Ruefa) [reply openapi.yaml]: I missed the securityDefinition. It is working now

**Pull request merged**